recipes/simmer/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "simmer" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7b6bb13189ebcec123e290163dce18d98c697c7fa9d8a78112cce2cdea53379e
+
+build:
+  number: 2
+  skip: True  # [py<36]
```

Comment:
I think we can drop the `skip` line. Also, I think this can probably be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Just use `python  >3.6` in both `host` and `run` in the `requirements` section.